react/features/video-menu/components/native/index.js

```diff
@@ -1,6 +1,5 @@
 // @flow
 
-export { default as BlockAudioVideoDialog } from './BlockAudioVideoDialog';
```

Comment:
Why are we removing this?

Reply:
It is not used anywhere.

Reply:
True that, you can remove the entire implementation, actually.